api.json

```diff
@@ -0,0 +1,732 @@
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "invalid block level": {
+                                        "$ref": "#/components/examples/invalid-block-level-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/blocks/head": {
+            "get": {
+                "description": "Returns the last applied block.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Current block retrieved.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Block"
+                                },
+                                "examples": {
+                                    "block not found": {
+                                        "$ref": "#/components/examples/block-not-found-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/chain/blocks/genesis": {
+            "get": {
+                "description": "Returns the genesis block of the chain.",
+                "tags": [
+                    "chain"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Genesis block has been retrieved.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Block"
+                                },
+                                "examples": {
+                                    "block not found": {
+                                        "$ref": "#/components/examples/block-not-found-error"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/consensus": {
+            "post": {
+                "description": "nodes related stuff, should only be used by nodes",
+                "tags": [
+                    "consensus"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Consensus operation accepted."
+                    }
+                }
+            }
+        },
+        "/consensus/operations": {
```

Comment:
Minor, this is not a consensus operation, so probably only `/operations`

Major, this needs to be done in two steps, because they need to sign a message for us, as discussed with @d4hines the idea is to have an endpoint like "/make-operation" which will return you some bytes that you should sign and then submit the bytes + signature to "/operations".

Look at how Tezos does.